app/models/ability.rb

```diff
@@ -112,9 +112,7 @@ def initialize(user)
     can :live_data, Cad, visible: true, public: true
     can :read, Scoreboard, public: true
 
-    can :read, [FindOutMore, Observation, TransportSurvey, TransportSurvey::Response, ProgrammeType]
-
-    can [:show, :index], SchoolTarget
```

Comment:
Just checking it is right to replace a :show, :index for SchoolTarget with just a :read ?

Reply:
Yes `:read` means `:show, :index`. One thing we can decide in tidying this further is to allow short hands like `:read` and `:manage` or just list all permissions.

Reply:
Ahh that's handy to know and much tidier 👍